apps/ui/src/components/IndicatorVotingPower.vue

```diff
@@ -1,40 +1,37 @@
 <script setup lang="ts">
-import { _vp } from '@/helpers/utils';
+import { utils } from '@snapshot-labs/sx';
+import { getFormattedVotingPower } from '@/helpers/utils';
 import { evmNetworks } from '@/networks';
 import { VotingPower, VotingPowerStatus } from '@/networks/types';
 import { NetworkID } from '@/types';
 
 const props = defineProps<{
   networkId: NetworkID;
-  status: VotingPowerStatus;
-  votingPowerSymbol: string;
-  votingPowers: VotingPower[];
+  votingPower?: {
+    totalVotingPower: bigint;
+    votingPowers: VotingPower[];
+    status: VotingPowerStatus;
+    symbol: string;
+    decimals: number;
+    error: utils.errors.VotingPowerDetailsError | null;
+  };
```

Comment:
Let's use `VotingPowerItem` type from the store.

Reply:
Should we export the type from the store (none of the stores are currently exporting types), or move that type to the types.ts file ?